packages/edit-site/src/components/header-edit-mode/style.scss

```diff
@@ -145,14 +145,24 @@ $header-toolbar-min-width: 335px;
 	.block-editor-block-mover {
 		border-left: none;
 
+		// Modified group borders
```

Comment:
This comment doesn't seem to be related to the actual pseudo-element style, so how about deleting this comment?